### ISSUE TITLE
Missing arraySlice operation for Polygon minimumHeights and maximumHeights

### ISSUE THREAD

**Issue by yaozhihang** [AnalyticalGraphicsInc/cesium#7396]
Hi, I think the following code 
```
minimumHeights = arraySlice(minimumHeights);
maximumHeights = arraySlice(maximumHeights);
```
should be added above the code line https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Vector3DTilePolygons.js#L288

**Comment by hpinkos** [comment]
@yaozhihang is there a bug that making this change fixes?

**Comment by yaozhihang** [comment]
Please find attached the test 3DTiles dataset. The polygon object cannot be rendered without the two additional code lines. 
[Test3DTilesVectorPolygon.zip](https://github.com/AnalyticalGraphicsInc/cesium/files/2663858/Test3DTilesVectorPolygon.zip)

**Comment by hpinkos** [comment]
@lilleyse @bagnell thoughts?

**Comment by lilleyse** [comment]
@yaozhihang your change looks right to me. `loadCRN` is doing something similar for array buffer views: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/loadCRN.js#L79.

Would you like to open a pull request with the fix?

**Comment by hpinkos** [comment]
Fixed in #7589
